numba/tests/test_intwidth.py

```diff
@@ -16,12 +16,7 @@ def usecase_uint64_constant():
     return 18446744073709551615
 
 def usecase_uint64_floor():
```

Comment:
It's just a tiny nit, but I think the name of the test case is obsolete now. Maybe `uint64_function`, `uint64_max` or similar?